actions/actions.py

```diff
@@ -399,14 +422,30 @@ def validate_amount_of_money(
 
     def submit(self, dispatcher, tracker, domain):
         if tracker.get_slot("confirm"):
+            amount_of_money = float(tracker.get_slot("amount_of_money"))
+            account_balance = float(tracker.get_slot("account_balance"))
+
+            if account_balance < amount_of_money:
+                dispatcher.utter_message(template="utter_insufficient_funds")
+                return [
+                    SlotSet("PERSON", None),
+                    SlotSet("amount_of_money", None),
+                    SlotSet("confirm", None),
+                ]
```

Comment:
The insufficient funds check should move to a slot validation function

Reply:
Cool, moved the insufficient funds check to the validation function for the transfer form and added an insufficient funds check to the CC_pay validation function as well.